test/zdtm/static/maps00.c

```diff
@@ -158,7 +158,8 @@ static int check_map(struct map *map)
 
 	if (!sigsetjmp(segv_ret, 1)) {
 		if (map->prot & PROT_WRITE) {
-			memcpy(map->ptr, test_func, getpagesize());
+			memcpy(map->ptr, test_func, ONE_MAP_SIZE);
```

Comment:
Why the change from `getpagesize()` to `ONE_MAP_SIZE`?

Reply:
I find `mmap(ONE_MAP_SIZE)` instead of `mmap(getpagesize())`. `getpagesize() == ONE_MAP_SIZE` isn't always correct, changing it to prevent the potential overflow.

Reply:
Looking at the code of the test I agree. It is strange that `getpagesize()` is used here.

It does not really fit the commit description, but splitting in two one-line commits seems unnecessary. I am okay with this.